dashboard/src/features/tutorial/components/TutorialModal/TutorialModal.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import styles from './TutorialModal.scss'
+import { Link } from 'react-router'
+
+class TutorialModal extends React.Component {
+
+  render() {
+    return (
+      <div className={styles.main}>
+        <div className={styles.backdrop} onClick={this.props.dismissTutorial}></div>
+          <div className={styles.content}>
+            <div className={styles.header}>
+              {this.props.title}
+            </div>
+            <div className={styles.text}>
+              {this.props.content.map(function (contentLine, i){
+                if(contentLine['list']){
+                  let list = []
+                  contentLine['list'].forEach(function(listItem, j){
+                    list.push(<tr key={j} className={styles.listItemGroup}>
+                      <td className={styles.listBullet}>{j+1}</td>
+                      <td>{listItem}</td>
+                    </tr>)
+                  })
+                  return <table className={styles.listItemContainer}>{list}</table>
+                } else {
+                  let value = contentLine
+                  if (typeof(value) === 'object') { value = value['line'] }
+                  return <p key={i}>{value}</p>
+                }
+              })}
+            </div>
+            <div className={styles.footer}>
+              <button onClick={this.props.dismissTutorial} className={`btn btn-primary ${styles.dismiss}`}>{this.props.dismiss}</button>
+              {this.props.button && <Link to={this.props.route}>
+                  <button key='showNext' className={`btn btn-primary ${styles.next}`} onClick={this.props.handleNext}>
+                    {this.props.button}
+                  </button>
+                </Link>}
+            </div>
+          </div>
+      </div>
+    )
+  }
+}
+
+import { actions } from 'features/tutorial'
+import { connect } from 'react-redux'
+
+const mapStateToProps = (state) => ({
+  tutorialRoute: state.tutorial.route,
+  currentStep: state.tutorial.currentStep,
+  showTutorial: state.routing.locationBeforeTransitions.pathname.startsWith(state.tutorial.route)
+})
+
+const mapDispatchToProps = ( dispatch ) => ({
+  dismissTutorial: () => dispatch(actions.dismissTutorial)
```

Comment:
You may want to double-check with @dominic, but I feel like one of the takeaways from our refactoring conversations is that we should be moving away from trivial action variables like `dismissTutorial`, and simply inline an action object such as `{type: 'DISMISS_TUTORIAL'}`. The rationale there is that you can search for the action string right away; as the code stands, you first have to look up `dismissTutorial` in order to figure out what reducer ultimately deals with this action.